instagram/models.py

```diff
@@ -10,7 +12,13 @@ def object_from_dictionary(cls, entry):
         return cls(**entry_str_dict)
 
     def __repr__(self):
-        return unicode(self).encode('utf8')
+        return str(self)
+
+    def __str__(self):
+        if six.PY3:
+            return self.__unicode__()
```

Comment:
There is no `__unicode__` method.

Reply:
@homm Every class in this file has it, it was there before these changes.